.github/workflows/test.yaml

```diff
@@ -74,22 +74,31 @@ jobs:
           comment_mode: off
   save-cache:
     name: Save Cache
-    runs-on: ubuntu-22.04
+    strategy:
+      matrix:
+        os: [ubuntu-22.04, windows-2022]
```

Comment:
Nit (multiple instances): prefer the following style for consistency:

```yaml
os:
  - ubuntu-22.04
  - windows-2022
```

The principal benefits of this style are that it makes diffs that modify the elements of a list more readable and also makes it simpler to write regexes that match a specific line (e.g., to configure tools like Renovate).

Reply:
I tried this syntax: 
```
runs-on:
      os:
        - ubuntu-22.04
        - windows-2022
```
and got the following error: 
 "runs-on" section must be sequence node but got mapping node with "!!map" tag [syntax-check]
   |
79 |       os:

Is there a different way I should try it?

Reply:
[You are missing the `strategy` and `matrix` keys and should remove `runs-on`](https://docs.github.com/en/actions/using-jobs/using-a-matrix-for-your-jobs).

Reply:
I tried removing `runs-on` and VS Code was unhappy with me--it had an error saying `Missing property "runs-on".yaml-schema: GitHub Workflow`.  Just to make sure, I ran it through MegaLinter, which also agreed.  When I try it with the following syntax:
```
strategy:
      matrix:
        os:
          - ubuntu-22.04
          - windows-2022
 runs-on: ${{ matrix.os }}
```
both VS Code and MegaLinter seem to be happy. I got that syntax [here](https://docs.github.com/en/actions/using-jobs/using-a-matrix-for-your-jobs#example-using-a-multi-dimension-matrix). It says it's for a multi-dimension matrix but it seems to apply to me.  Do you think it's ok?